crates/miden-lib/asm/account_components/rpo_falcon_512_procedure_acl.masm

```diff
@@ -110,7 +112,7 @@ export.auth__tx_rpo_falcon512_procedure_acl.2
 
     # If authentication is required, perform signature verification
     if.true
-        exec.::miden::contracts::auth::basic::auth__tx_rpo_falcon512
+        exec.::miden::auth::rpo_falcon512::authenticate_transaction
```

Comment:
Related to the previous comment: if we passed `PUB_KEY` in as a parameter, the storage layout of this component  would not need to be implicitly tied to the storage layout of the simple RPO Falcon512 component.